google-clis-common/src/lib.rs

```diff
@@ -125,9 +125,9 @@ impl AsRef<str> for CallType {
 #[derive(Clone, Default)]
 pub struct FieldCursor(Vec<String>);
 
-impl ToString for FieldCursor {
-    fn to_string(&self) -> String {
-        self.0.join(".")
+impl Display for FieldCursor {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "{}", self.0.join("."))
```

Comment:
This could also be `self.0.join(".").fmt(f)` I think. In case you like it more, that could be changed in the other PR of yours.

Reply:
That is true, I don't really care too much I'm just used to the `write` macro way but both ways are fine with me.